src/ir_Kelvinator.cpp

```diff
@@ -1,18 +1,17 @@
 // Copyright 2016 David Conran
 //
-// Code to emulate IR Kelvinator YALIF remote control unit, which should control
-// at least the following Kelvinator A/C units:
-// KSV26CRC, KSV26HRC, KSV35CRC, KSV35HRC, KSV53HRC, KSV62HRC, KSV70CRC,
-// KSV70HRC, KSV80HRC.
-//
-// Note:
-// * Unsupported:
-//    - All Sleep modes.
-//    - All Timer modes.
-//    - "I Feel" button & mode.
-//    - Energy Saving mode.
-//    - Low Heat mode.
-//    - Fahrenheit.
+/// Code to emulate IR Kelvinator YALIF remote control unit, which should
+/// control at least the following Kelvinator A/C units:
+/// KSV26CRC, KSV26HRC, KSV35CRC, KSV35HRC, KSV53HRC, KSV62HRC, KSV70CRC,
+/// KSV70HRC, KSV80HRC.
```

Comment:
this is already in the .h file, maybe cleanup instead?

Reply:
In a later PR perhaps. We can discuss where we should put comments like these to be more consistent. I originally thought putting things in the header files would be best as most people might look there first (I would), but now with `doxygen` doing the documentation, it might pay to put things in the `.cpp` files so it's consistent with protocols that don't have header files, and that it doesn't really matter as much (with doxygen)